packages/react/src/Testimonial/Testimonial.tsx

```diff
@@ -27,14 +27,19 @@ export type TestimonialProps = {
    * Sets the testimonial text size
    */
   size?: TestimonialSize
+  /** Sets the color of the quote mark */
+  quoteMarkFill?: React.CSSProperties['color']
```

Comment:
I used the word "Fill" incase we decide to allow gradient or image fills in the future.